symplyphysics/laws/thermodynamics/irradiation_of_black_body_from_temperature.py

```diff
@@ -8,24 +8,24 @@
 ## power of the black body's thermodynamic temperature T (also called absolute temperature). 
 
 # Law: j* = sigma*T^4, where
-## j* is irradiance,
+## j* is radiant heat energy,
 ## sigma is constant of proportionality,called the Stefan–Boltzmann constant,
 ## T is temperature of a completely black body
 
 # Note
 ## j* = epsilon*sigma*T^4, where ε is the integral absorption capacity of the body. For a completely black body ε = 1.
 
-irradiance = Symbol("irradiance", units.power/units.length**2)
-temperature = Symbol("temperature", units.temperature)
+irradiance = Symbol("irradiance", units.watt/units.meter**2)
```

Comment:
It's wrong. Those are SI units, not dimensions. There are other, non-SI unit systems, therefore we should not write laws in SI terms.